src/features/invoiceCard/InvoiceCardContainer.tsx

```diff
@@ -48,17 +48,19 @@ const InvoiceCardContainer = ({
     setSelectedInvoiceAction(null);
   };
 
-  const actionsDisabled = !(order?.status === OrderStatus.ERROR || order?.status === OrderStatus.OFFERED);
-
   return (
     <>
       <InvoiceActions
-        disabled={actionsDisabled}
         selectedAction={selectedInvoiceAction}
         actions={[
           {
             value: 0,
             label: t('invoiceCard.markAsPaid.label'),
+            disabled: !(
+              order?.status === OrderStatus.ERROR ||
+              order?.status === OrderStatus.OFFERED ||
+              order?.status === OrderStatus.DRAFTED
```

Comment:
Is there a reason that this is duplicated in `invoicingDisabled`? This could probably be extracted

Reply:
"Mark as paid" would change the status to be paid manually while the "invoicing disabled" is for sending an offer, aka it changes the status to be "offered" and these conditions happen to be the same for both but that won't be necessarily the case, therefore, I didn't extract them to one variable.